### ISSUE TITLE
Escape character should close Attendance UI modal

### ISSUE THREAD

**Issue by ghost** [WordPress/wordcamp.org#621]
Imported from https://meta.trac.wordpress.org/ticket/4375 
Created by @iandunn:

After clicking on an attendee's name, a modal pops up showing their avatar and buttons to set whether or not they attended.

To close that modal, you have to click on the `x`. You should also be able to close it by hitting the `escape` key.

**Comment by ghost** [comment]
[Comment](https://meta.trac.wordpress.org/ticket/4375#comment:1) by slackbot:

_This ticket was mentioned in [Slack](https://make.wordpress.org/chat/) in #meta-wordcamp by iandunn. [View the logs](https://wordpress.slack.com/archives/meta-wordcamp/p1594314290051300)._

**Comment by ryelle** [comment]
If anyone wants to take this on from an accessibility perspective, the dialog should probably also trap focus until it's closed, and closing should reset the focus back to the clicked attendee.
